pytorch/training/docker/1.9/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,146 @@
+FROM ubuntu:20.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG PYTHON=python3
+# Add arguments to achieve the version, python and url
+ARG PYTHON_VERSION=3.8.5
+ARG OPEN_MPI_VERSION=4.0.1
+
+# The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
+# the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
+ARG SMDEBUG_VERSION=1.0.9
```

Comment:
@leleamol Could you please confirm?

Reply:
I have tested the torch binary with 1.0.9 and it has passed the basic tests. Hence so far, we are good with this version.
In case, we need to release new smdebug and this PR is merged,  the smdebug team will create a separate PR.